internal/scan-manager/core/pom.xml

```diff
@@ -168,6 +183,15 @@
                     <path>/</path>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <suiteXmlFiles>
+                        <suiteXmlFile>src/test/resources/testng.xml</suiteXmlFile>
+                    </suiteXmlFiles>
+                </configuration>
+            </plugin>
```

Comment:
Shall we please add `maven-java-doc` plugin to this project (`core`) with this PR. Add the commented configuration at https://github.com/wso2/security-tools/blob/master/internal/pom.xml#L264

We can incrementally add it to other projects.